vercel.json

```diff
@@ -1,17 +0,0 @@
-{
-  "headers": [
-    {
-      "source": "/(.*)",
-      "headers" : [
-        {
-          "key" : "X-Frame-Options",
-          "value" : "DENY"
-        },
-        {
-          "key" : "Content-Security-Policy",
-          "value" : "frame-ancestors 'none'"
```

Comment:
If you want to allow embedding the website now, we should also removed the protection against embedding at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/3b8fe1626247afa21f8c0710ebbffb0340007776/server.js#L50-L51.

Reply:
@norbertschuler server.js is only used for heroku deployments, not vercel correct? 

@sagararyal Should this be changed for heroku as well?

Reply:
I have to check Vercel, Heroku and Netlify for the build command and configuration to see where we use a static build and where we use some SSR.

Reply:
@norbertschuler This is done